### PR TITLE
HOTFIX: Makes Sentient Disease actually playable (150 hours certified)

### LINKED ISSUE #41499: Sentient Disease: No Symptoms on Spawn
Issue reported from  Round ID: 97088 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])

Reporting client version: 512

A sentient disease randomly spawned during the round but they had no symptoms when they infected their first host. No cough on command or anything. They only had one evolution point which could not afford them anything. This made them nothing more than a spectator than an antag.

I spawned a sentient disease shortly after receiving the ahelp from the first one just to see if this wasn't some odd one time issue but it was consistent.

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
fixes #41499

### CODE REVIEW

**Comment by k-k-karma** [comment]
nice

**Review by ShizCalev — DISMISSED** [review]

File: code/modules/antagonists/disease/disease_abilities.dm [hunk 1]
```diff
@@ -5,6 +5,9 @@ is currently following.
 */
 
 GLOBAL_LIST_INIT(disease_ability_singletons, list(
+new /datum/disease_ability/action/cough(),
```
Comment (ShizCalev) [inline code/modules/antagonists/disease/disease_abilities.dm]: that's not a proc.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/disease/disease_abilities.dm [hunk 2]
```diff
@@ -5,6 +5,9 @@ is currently following.
 */
 
 GLOBAL_LIST_INIT(disease_ability_singletons, list(
```
Comment (ShizCalev) [inline code/modules/antagonists/disease/disease_abilities.dm]: honestly this should probably just be changed to a typecache.

Comment (ExcessiveUseOfCobblestone) [reply code/modules/antagonists/disease/disease_abilities.dm]: me can't read code, are typecaches the same as new type/1, new type/2, etc.

Comment (Cruix) [reply code/modules/antagonists/disease/disease_abilities.dm]: There is no reason for this to be a typecache.

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**